modules/virt-generalizing-linux-vm-image.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/creating_vms_custom/virt-creating-vms-uploading-images.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-generalizing-linux-vm-image_{context}"]
+= Generalizing a VM image
+
+You can generalize a {op-system-base-full} image to remove all system-specific configuration data before you use the image to create a golden image, a preconfigured snapshot of a virtual machine (VM). You can use a golden image to deploy new VMs.
+
+Generalizing a {op-system-base} VM is done using the `virtctl`, `guestfs`, and `virt-sysprep` tools.
+
+.Prerequisites
+
+* An {op-system-base} virtual machine.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the `virtctl` tool.
+
+.Procedure
+
+. If the {op-system-base} virtual machine is running, stop it.
++
+[source,terminal]
+----
+$ virtctl stop <rhel-vm-name>
+----
+
+. Optional: Clone the virtual machine to avoid losing the data from your original VM. You can then generalize the cloned VM.
```

Comment:
I'm going to leave this as is, since I think the alternation of virtual machine and VM (since they're mentioned 3 times) actually makes this easier for the user to parse.